charts/bookkeeper-operator/templates/operator.yaml

```diff
@@ -21,6 +21,10 @@ spec:
       - name: {{ template "bookkeeper-operator.fullname" . }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        {{- if .Values.securityContext }}
+        securityContext:
+{{ toYaml .Values.securityContext | indent 10 }}
```

Comment:
It would probably be better to have separate securityContext settings for the operator an for the hooks (they might be run as different users if a custom hook image is used)

Reply:
Made the changes as suggseted